BEIMA.Client/src/shared/NavBar.js

```diff
@@ -27,16 +32,22 @@ const NavBar = () => {
               <Row className="pageLink">
                 <Link to="/buildings" className="sharedText">Buildings</Link>
               </Row>
+              {currentUser.Role === 'admin' ?
```

Comment:
I'm wondering, should we only allow regular users to interact with just devices? Or do we still want to give them access to both buildings and device types, or one or the other?

Reply:
I think access to everything besides Users is fine. Maybe a user could be entering a device that is in a new building and will need to add the building as well. Same thing for device type